Packs/Cylance_Protect/Integrations/Cylance_Protect_v2/Cylance_Protect_v2.yml

```diff
@@ -265,6 +265,9 @@ script:
     - contextPath: File.SHA256
       description: The SHA256 hash of the threat.
       type: string
+    - contextPath: File.MD5
+      description: The SHA256 hash of the threat.
+      type: string
```

Comment:
Shouldn't new context paths need to be added to all commandsd using the new method? Is it already there?

Reply:
They are already there. The thing is that there are no File.Sha256 for any of the commands (the CamleCase version). I wasnt sure if if should add them, what do you think?

Reply:
They are already there. The thing is that there are no File.Sha256 for any of the commands (the CamleCase version). I wasnt sure if if should add them, what do you think?

Reply:
Add them, if the output paths don't match the real context then it's another bug that should be solved.